### PR TITLE
Support async measure function

### PR DESCRIPTION
**Pull request by ruowan**

### CODE REVIEW

File: src/measure.ts [hunk 1]
```diff
@@ -11,3 +11,15 @@ export function measure(target: any, propertyKey: string, descriptor: PropertyDe
 
   return descriptor;
 }
+
+export function measureAsync(target: any, propertyKey: string, descriptor: PropertyDescriptor): any {
```
Comment (NetanelBasal) [inline src/measure.ts:15]: add specs, please

Comment (ruowan) [reply src/measure.ts:15]: Hi, what's is specs?

Comment (ruowan) [reply src/measure.ts]: Added unittest.

**Comment by ruowan** [comment]
Some file changes is done by auto code formatter.

**Pull request merged**